examples/gitlab/gitlab.yml

```diff
@@ -1,18 +1,33 @@
 name: gitlab
+
 labels:
     app: gitlab
+
 replicas: 1
+
 containers:
 - name: gitlab
   image: gitlab/gitlab-ce:9.4.1-ce.0
   imagePullPolicy: ""
+  health:
+    httpGet:
+      path: /help
+      port: 80
+    initialDelaySeconds: 200
```

Comment:
readiness and liveliness probes originally had different `initalDelaySeconds`.

But I suppose that for purpose of the demo it is  ok to consolidate (maybe)
What @surajnarwade thinks about this ?